model-optimizer/mo/ops/softmax.py

```diff
@@ -87,7 +99,7 @@ def __init__(self, graph: Graph, attrs: dict):
             'axis': 1,
             'type': None,  # the operation will be replaced with a
                            # Reshape(LogSoftmax(FlattenONNX(x, axis), 1), x.shape) sub-graph
-            'op': __class__.op,
+            'op': self.op,
             'in_ports_count': 1,
             'out_ports_count': 1,
         }, attrs)
```

Comment:
To have onnx specific op in ops directory is not a good way. Why don't you just create a replacer for onxx node or so and use a generic op for all frameworks?

Reply:
We do that for many other ops too. Since you need to create op first in the extractor and only afterward replace with replacer.

Reply:
Yes, I understand but this does not mean a correct way. To have a generic operation here is more reasonable in terms of flexibility.